.github/workflows/ci.yml

```diff
@@ -0,0 +1,98 @@
+name: Alfresco REST API Explorer CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  push:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  workflow_dispatch:
+  schedule:
+    - cron: '0 5 * * 3'
+
+env:
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  RELEASE_VERSION: "7.3.0"
+  DEVELOPMENT_VERSION: "7.4.0-A1-SNAPSHOT"
+  MAVEN_CLI_OPTS: "-B -q -e -fae -V -DinstallAtEnd=true -U"
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 30
+
+jobs:
+  pre_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.34.2
+
+  build:
+    name: "Build + deploy"
```

Comment:
Why is this called "Build **+ deploy**"?

Reply:
Renamed to "Build".